pp/integrations/plugin-packs/procedures/operatingsystems-freebsd-snmp.md

```diff
@@ -20,28 +26,28 @@ The connector brings the following service templates (sorted by the host templat
-| Disk-Global          | OS-FreeBSD-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each check the mount point of disks will appear                            |     X     |
-| Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Check disk access. For each check the name of the disk will appear as a "label" rather than the letter assigned |           |
-| Inodes-Global        | OS-Freebsd-Inodes-Global-SNMP-custom        | Check Inodes space usage on partitions                                                                                 |           |
-| Process-Generic      | OS-FreeBSD-Process-Generic-SNMP-custom      | Check Unix process/service is working                                                                                  |           |
-| Traffic-Generic-Id   | OS-FreeBSD-Traffic-Generic-Id-SNMP-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear                            |           |
-| Traffic-Generic-Name | OS-FreeBSD-Traffic-Generic-Name-SNMP-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear                            |           |
-| Traffic-Global       | OS-FreeBSD-Traffic-Global-SNMP-custom       | Check the bandwidth of interfaces. For each checks the name of the interface will appear                               |     X     |
-| Uptime               | OS-FreeBSD-Uptime-SNMP-custom               | Time since the server has been working and available                                                                   |           |
+| Service Alias        | Service Template                            | Service Description                                                                                                    | Discovery  |
+|:---------------------|:--------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------|:----------:|
+| Disk-Generic-Id      | OS-FreeBSD-Disk-Generic-Id-SNMP-custom      | Check the rate of free space on the disk. For each checks the name of the disk will appear                             |            |
+| Disk-Generic-Name    | OS-FreeBSD-Disk-Generic-Name-SNMP-custom    | Check the rate of free space on the disk. For each checks the mount pont of the disk will appear                       |            |
+| Disk-Global          | OS-FreeBSD-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each checks the mount point of disks will appear                            | X          |
+| Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Check access disk of the disk. For each check the name of the disk will appear "label" rather than the letter assigned |            |
```

Comment:
```suggestion
| Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Check disk access. For each check the name of the disk will appear as a "label" rather than the letter assigned |            |
```